src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -65,4 +71,9 @@ private function getExpressionLanguage()
 
         return $this->expressionLanguage;
     }
+
+    private function getPropertyAccessor()
```

Comment:
IMO we don't need this method anymore and can inline the `createPropertyAccessor()` call instead.